tests/target-browser/target_browser_test.go

```diff
@@ -415,12 +416,52 @@ var _ = Describe("Target Browser Tests", func() {
 		})
 
 		It("Should get one page backup", func() {
-			isLast = true
 			args := []string{cmdGet, cmdBackup, flagBackupPlanUIDFlag, backupPlanUIDs[1], flagPageSize, "1"}
 			backupData := runCmdBackup(args)
 			Expect(len(backupData)).To(Equal(1))
 		})
 
+		It("Should get one backupPlan for specific backupPlan UID", func() {
+			args := []string{cmdGet, cmdBackupPlan, backupPlanUIDs[1]}
+			backupPlanData := runCmdBackupPlan(args)
+			Expect(len(backupPlanData)).To(Equal(1))
+			Expect(backupPlanData[0].BackupPlanUID).To(Equal(backupPlanUIDs[1]))
+		})
+
+		It("Should get two backupPlan for backupPlan UIDs", func() {
+			args := []string{cmdGet, cmdBackupPlan, backupPlanUIDs[0], backupPlanUIDs[1]}
+			backupPlanData := runCmdBackupPlan(args)
+			Expect(len(backupPlanData)).To(Equal(2))
```

Comment:
check for data as well like - whether same UIDs were returned which were requested.

Reply:
done